src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String BLANK = "    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcome();
+        Scanner in = new Scanner(System.in);
+        String inputString;
+        ArrayList<Task> taskList = new ArrayList<>();
+        int numberOfTasks = 0;
+
+        while (true) {
+            try {
+                inputString = in.nextLine();
+                String[] command = inputString.split(" ", 2);
+
+                switch (command[0]) {
+                case "list":
```

Comment:
Perhaps better to use constants instead of strings. For example, COMMAND_LIST instead of "list"